understanding/20/audio-control.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Audio Control</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>A page that plays music or sounds doesn't disrupt people</dd>
-		<dt>Author task</dt><dd>If you play audio content automatically, let people turn it down or off</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>A page that plays music or sounds doesn't disrupt people.</dd>
+         <dt>What to do</dt><dd>If you play audio content automatically, let people turn it down or off.</dd>
+         <dt>Why it's important</dt><dd>Sound distracts some people, and also interferes with screen readers.</dd> 
```

Comment:
@rainbreaw commented
> There are actually times that the intention is to disrupt, or take the user's attention away from something and direct it to something else out of urgency.
> Recent example from my own life: local tornado warning with immediate threat to life and property. Can this edit be re-written to acknowledge that this is only meant for unnecessary times?
> Another possible example: the individual is reaching a time limit and needs to be redirected to options to extend or mitigate the effects of that time limit.

Reply:
The Audio control SC is specifically about  content that "plays automatically for more than 3 seconds" and the requirement is to be able to pause or stop the audio, or control its volume.

The scenario you are referring to, where audio is intentionally used to alert people to a danger, does not appear to be mentioned or addressed in the Understanding document. It may be an idea to make a PR to add in this material, but the In brief section should not introduce new concepts. Its purpose is to highlight things covered in more detail in the Understanding document.

I will offer that the alerts/warnings you are talking about, IMO, can still be achieved within the confines of the requirement. Obviously we don't want any 'silence' mechanism (as described in the Understanding example) to suppress emergency warnings, and that could be captured in a note.

Reply:
@mbgower thank you for this further explanation. Based on this, I'm comfortable with the PR as written.